packages/analytics-types/src/config/browser.ts

```diff
@@ -33,14 +33,9 @@ export interface DefaultTrackingOptions {
 }
 
 export interface TrackingOptions {
-  deviceManufacturer?: boolean;
```

Comment:
Do we still want consumers to be able to set these values by themselves?

Reply:
Yes, they can set `event.device_manufacturer` through an enrichment plugin. This option was only used to enable client side user agent parsing which we no longer have.